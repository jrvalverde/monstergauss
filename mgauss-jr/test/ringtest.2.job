$JOB J.C9H15.1 GAU,1080 S SLOF=O.C9H15 TIME=10
$AS 7 TO Z.C9H15.1
$AS 8 TO 8.C9H15 BLOC=N 
$RUN GAUSS
*MOL BA=M3, GOPT, ME=OC, RING, RESTART, GU=N
*PRT P=Y, G=Y 
*PUN Z
*IOP
  18  26   2  SET ANGLE / DIHEDRAL ANGLE CUTOFF BOND LENGTH TO 2.0 ANG
  
H+ EXOCYCLIC METHYLENE CIS-FUSED [2,4]-BICYCLOOCTANE GOPT, STO-3G 
 1 1
C1
C2     C1        C1C2 
C3     C2        C2C3       C1        C3C2C1
C4     C3        C3C4       C2        C4C3C2     C1        TAUC4       0
C5     C1        C1C5       C2        C5C1C2     C3        TAUC5       0
C6     C2        C2C6       C1        C6C2C1     C3        TAUC6       0
C7     C6        C6C7       C2        C7C6C2     C1        TAUC7       0
C8     C1        C1C8       C2        C8C1C2     C5        TAUC8       0
C9     C8        C8C9       C1        C9C8C1     C2        TAUC9       0
H10    C1        C1H10      C2        H10C1C2    C5        TAUH10      0
H11    C2        C2H11      C1        H11C2C1    C3        TAUH11      0
H12    C4        C4H12      C3        H12C4C3    C2        TAUH12      0
H13    C4        C4H13      C3        H13C4C3    H12       TAUH13      0
H14    C5        C5H14      C1        H14C5C1    C2        TAUH14      0
H15    C5        C5H15      C1        H15C5C1    C2        TAUH15      0
H16    C6        C6H16      C2        H16C6C2    C7        TAUH16      0
H17    C6        C6H17      C2        H17C6C2    C7        TAUH17      0
H18    C7        C7H18      C6        H18C7C6    C2        TAUH18      0
H19    C7        C7H19      C6        H19C7C6    C2        TAUH19      0
H20    C8        C8H20      C1        H20C8C1    C9        TAUH20      0
H21    C8        C8H21      C1        H21C8C1    C9        TAUH21      0
H22    C9        C9H22      C8        H22C9C8    C1        TAUH22      0
H23    C9        C9H23      C8        H23C9C8    C1        TAUH23      0
H24    C4        C4H24      C3        H24C4C3    H12       TAUH24      0
  
C1C2     =    1.5621510 
C2C3     =    1.5342011 
C3C4     =    1.500 
C1C5     =    1.5589988 
C2C6     =    1.5438670 
C6C7     =    1.5469656 
C1C8     =    1.5409937 
C8C9     =    1.5469093 
C1H10    =    1.0882362 
C2H11    =    1.0900185 
C4H12    =    1.0810589 
C4H13    =    1.0811272 
C5H14    =    1.0883779 
C5H15    =    1.0885495 
C6H16    =    1.0878663 
C6H17    =    1.0872356 
C7H18    =    1.0876653 
C7H19    =    1.0875831 
C8H20    =    1.0879860 
C8H21    =    1.0872523 
C9H22    =    1.0876677 
C9H23    =    1.0875577 
C4H24    =    1.0811272 
C3C2C1   =     88.76995 
C4C3C2   =    133.97149 
C5C1C2   =     90.07246 
C6C2C1   =    113.37976 
C7C6C2   =    110.27816 
C8C1C2   =    113.05726 
C9C8C1   =    110.52317 
H10C1C2  =    112.83402 
H11C2C1  =    113.19807 
H12C4C3  =    106.0 
H13C4C3  =    110.0 
H14C5C1  =    114.59449 
H15C5C1  =    114.71224 
H16C6C2  =    110.22623 
H17C6C2  =    109.76999 
H18C7C6  =    108.61941 
H19C7C6  =    108.90386 
H20C8C1  =    110.28282 
H21C8C1  =    109.69541 
H22C9C8  =    108.64871 
H23C9C8  =    108.91238 
H24C4C3  =    110.0 
TAUC4    =    179.67951 
TAUC5    =       .17570 
TAUC6    =    119.36025 
TAUC7    =     52.00101 
TAUC8    =   -119.70064 
TAUC9    =    -51.85610 
TAUH10   =    114.99383 
TAUH11   =   -114.91963 
TAUH12   =    -90.0 
TAUH13   =    120.0 
TAUH14   =    116.45500 
TAUH15   =   -116.92767 
TAUH16   =    121.19786 
TAUH17   =   -120.82722 
TAUH18   =     69.69928 
TAUH19   =    185.70759 
TAUH20   =   -121.31951 
TAUH21   =    120.74244 
TAUH22   =    -69.50781 
TAUH23   =    174.47749 
TAUH24   =   -120.0 
  
 0 0 0
 0 0 0 0
 100   1   0
  
 0 1 0
 0 2 0
 0 3 0
 0 5 0
 0 6 0
 0 7 0
 0 8 0
 0 9 0
 0 0 0
 0 1 2 0
 0 1 5 0
 0 2 3 0
 0 3 5 0
 0 2 6 0
 0 6 7 0
 0 7 9 0
 0 8 9 0
 0 1 8 0
 0 0 0 0
$EOJ
$$
