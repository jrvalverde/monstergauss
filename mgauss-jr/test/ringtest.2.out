 WELCOME TO U OF T MONSTERGAUSS                                                                         UNX VERSION: 13 AUG 1991
 ******************************
0CONTROL CARDS:

 *MOL BA=M3, GOPT, ME=OC, RING, RESTART, GU=N                            
 *PRT P=Y, G=Y                                                           
 *PUN Z                                                                  
 *IOP                                                                    
   18  26   2  SET ANGLE / DIHEDRAL ANGLE CUTOFF BOND LENGTH TO 2.0 ANG  
    0   0   0                                                            

0SEQUENCE    GROUP     MG OPTIONS         GROUP OPTIONS (USER CONTROLLABLE)                  LINKS TO BE CALLED          JUMP

     1         2      000000010300 000001000000000000000000000000000000000000000000   01000000000000000000000000000000     0
     2         3      000000010300 304000000000000000000000000000000000000000000000   11000100000000000000000000000000     0
     3         4      000000010300 500000000000000010000000000000000000000000000000   01000000000000000000000000000000     0
     4         5      000000010300 400000000000000000000000000000000000000000000000   10000000000000000000000000000000     0
     5         6      000000010300 001001000000000000001000000000000000000000000000   10000000000000000000000000000000     0
     8         7      000000010300 221000101000100000000000000000000000000000000000   01100000001000000000000000000000     0
    10         3      000000010300 004000000010000000000000000000000000000000000000   01000100000000000000000000000000     0
    11         5      000000010300 400000001000000000000000000000000000000000000000   10000000000000000000000000000000     0
    15         7      000000010300 121000101000000000000000000000000000000000000000   01100000001000000000000000000000    -2
    18         6      000000010300 001001000000021000000000000000000000000000000000   10000000000000000000000000000000     0
0OPTION NUMBERS:      000000000111 111111122222222223333333333444444444455555555556   66666666677777777778888888888999
                      123456789012 345678901234567890123456789012345678901234567890   12345678901234567890123456789012

0CONVERSION FACTORS:  1 BOHR = 0.52917706 ANGSTROMS
0                     1 HARTREE/BOHR = 8.2388584 MDYNE

                      1 HARTREE/ANGSTROM**2 = 4.3598149 MDYNE/ANGSTROM

                      1 ELECTRON*BOHR = 2.5417655 DEBYE

                      1 HARTREE/PARTICLE = 627.50959 KCAL/MOL = 2625.5001 KJ/MOL = 27.211611 EV/PARTICLE = 219474.67 CM(-1)/PARTICLE

                      1 ATOMIC UNIT OF ELECTRIC FIELD = 5.1422506E+09 VOLTS/CM
1H+ EXOCYCLIC METHYLENE CIS-FUSED [2,4]-BICYCLOOCTANE GOPT, STO-3G       
 ------------------------------------------------------------------------
0CHARGE  =   1          MULTIPLICITY  =  1

0FREE FORMAT Z MATRIX:

 C1                                                                      
 C2     C1        C1C2                                                   
 C3     C2        C2C3       C1        C3C2C1                            
 C4     C3        C3C4       C2        C4C3C2     C1        TAUC4       0
 C5     C1        C1C5       C2        C5C1C2     C3        TAUC5       0
 C6     C2        C2C6       C1        C6C2C1     C3        TAUC6       0
 C7     C6        C6C7       C2        C7C6C2     C1        TAUC7       0
 C8     C1        C1C8       C2        C8C1C2     C5        TAUC8       0
 C9     C8        C8C9       C1        C9C8C1     C2        TAUC9       0
 H10    C1        C1H10      C2        H10C1C2    C5        TAUH10      0
 H11    C2        C2H11      C1        H11C2C1    C3        TAUH11      0
 H12    C4        C4H12      C3        H12C4C3    C2        TAUH12      0
 H13    C4        C4H13      C3        H13C4C3    H12       TAUH13      0
 H14    C5        C5H14      C1        H14C5C1    C2        TAUH14      0
 H15    C5        C5H15      C1        H15C5C1    C2        TAUH15      0
 H16    C6        C6H16      C2        H16C6C2    C7        TAUH16      0
 H17    C6        C6H17      C2        H17C6C2    C7        TAUH17      0
 H18    C7        C7H18      C6        H18C7C6    C2        TAUH18      0
 H19    C7        C7H19      C6        H19C7C6    C2        TAUH19      0
 H20    C8        C8H20      C1        H20C8C1    C9        TAUH20      0
 H21    C8        C8H21      C1        H21C8C1    C9        TAUH21      0
 H22    C9        C9H22      C8        H22C9C8    C1        TAUH22      0
 H23    C9        C9H23      C8        H23C9C8    C1        TAUH23      0
 H24    C4        C4H24      C3        H24C4C3    H12       TAUH24      0
                                                                         
0VARIABLES (OPTIMIZED ON GOPT RUNS):
 C1C2     =    1.5621510                                                 
 C2C3     =    1.5342011                                                 
 C3C4     =    1.500                                                     
 C1C5     =    1.5589988                                                 
 C2C6     =    1.5438670                                                 
 C6C7     =    1.5469656                                                 
 C1C8     =    1.5409937                                                 
 C8C9     =    1.5469093                                                 
 C1H10    =    1.0882362                                                 
 C2H11    =    1.0900185                                                 
 C4H12    =    1.0810589                                                 
 C4H13    =    1.0811272                                                 
 C5H14    =    1.0883779                                                 
 C5H15    =    1.0885495                                                 
 C6H16    =    1.0878663                                                 
 C6H17    =    1.0872356                                                 
 C7H18    =    1.0876653                                                 
 C7H19    =    1.0875831                                                 
 C8H20    =    1.0879860                                                 
 C8H21    =    1.0872523                                                 
 C9H22    =    1.0876677                                                 
 C9H23    =    1.0875577                                                 
 C4H24    =    1.0811272                                                 
 C3C2C1   =     88.76995                                                 
 C4C3C2   =    133.97149                                                 
 C5C1C2   =     90.07246                                                 
 C6C2C1   =    113.37976                                                 
 C7C6C2   =    110.27816                                                 
 C8C1C2   =    113.05726                                                 
 C9C8C1   =    110.52317                                                 
 H10C1C2  =    112.83402                                                 
 H11C2C1  =    113.19807                                                 
 H12C4C3  =    106.0                                                     
 H13C4C3  =    110.0                                                     
 H14C5C1  =    114.59449                                                 
 H15C5C1  =    114.71224                                                 
 H16C6C2  =    110.22623                                                 
 H17C6C2  =    109.76999                                                 
 H18C7C6  =    108.61941                                                 
 H19C7C6  =    108.90386                                                 
 H20C8C1  =    110.28282                                                 
 H21C8C1  =    109.69541                                                 
 H22C9C8  =    108.64871                                                 
 H23C9C8  =    108.91238                                                 
 H24C4C3  =    110.0                                                     
 TAUC4    =    179.67951                                                 
 TAUC5    =       .17570                                                 
 TAUC6    =    119.36025                                                 
 TAUC7    =     52.00101                                                 
 TAUC8    =   -119.70064                                                 
 TAUC9    =    -51.85610                                                 
 TAUH10   =    114.99383                                                 
 TAUH11   =   -114.91963                                                 
 TAUH12   =    -90.0                                                     
 TAUH13   =    120.0                                                     
 TAUH14   =    116.45500                                                 
 TAUH15   =   -116.92767                                                 
 TAUH16   =    121.19786                                                 
 TAUH17   =   -120.82722                                                 
 TAUH18   =     69.69928                                                 
 TAUH19   =    185.70759                                                 
 TAUH20   =   -121.31951                                                 
 TAUH21   =    120.74244                                                 
 TAUH22   =    -69.50781                                                 
 TAUH23   =    174.47749                                                 
 TAUH24   =   -120.0                                                     
1
 GEOMETRY RESTARTED FROM DISK FILE

0---------------------------------------------------------------------------------------------------
                                             Z MATRIX
0  I   AN   Z1      BL                  Z2     ALPHA                Z3      BETA                Z4
 ---------------------------------------------------------------------------------------------------
   1    6
   2    6    1   0.0000000   (  1)   
   3    6    2   0.0000000   (  2)       1     0.00000   ( 24)   
   4    6    3   0.0000000   (  3)       2     0.00000   ( 25)       1     0.00000   ( 46)       0
   5    6    1   0.0000000   (  4)       2     0.00000   ( 26)       3     0.00000   ( 47)       0
   6    6    2   0.0000000   (  5)       1     0.00000   ( 27)       3     0.00000   ( 48)       0
   7    6    6   0.0000000   (  6)       2     0.00000   ( 28)       1     0.00000   ( 49)       0
   8    6    1   0.0000000   (  7)       2     0.00000   ( 29)       5     0.00000   ( 50)       0
   9    6    8   0.0000000   (  8)       1     0.00000   ( 30)       2     0.00000   ( 51)       0
  10    1    1   0.0000000   (  9)       2     0.00000   ( 31)       5     0.00000   ( 52)       0
  11    1    2   0.0000000   ( 10)       1     0.00000   ( 32)       3     0.00000   ( 53)       0
  12    1    4   0.0000000   ( 11)       3     0.00000   ( 33)       2     0.00000   ( 54)       0
  13    1    4   0.0000000   ( 12)       3     0.00000   ( 34)      12     0.00000   ( 55)       0
  14    1    5   0.0000000   ( 13)       1     0.00000   ( 35)       2     0.00000   ( 56)       0
  15    1    5   0.0000000   ( 14)       1     0.00000   ( 36)       2     0.00000   ( 57)       0
  16    1    6   0.0000000   ( 15)       2     0.00000   ( 37)       7     0.00000   ( 58)       0
  17    1    6   0.0000000   ( 16)       2     0.00000   ( 38)       7     0.00000   ( 59)       0
  18    1    7   0.0000000   ( 17)       6     0.00000   ( 39)       2     0.00000   ( 60)       0
  19    1    7   0.0000000   ( 18)       6     0.00000   ( 40)       2     0.00000   ( 61)       0
  20    1    8   0.0000000   ( 19)       1     0.00000   ( 41)       9     0.00000   ( 62)       0
  21    1    8   0.0000000   ( 20)       1     0.00000   ( 42)       9     0.00000   ( 63)       0
  22    1    9   0.0000000   ( 21)       8     0.00000   ( 43)       1     0.00000   ( 64)       0
  23    1    9   0.0000000   ( 22)       8     0.00000   ( 44)       1     0.00000   ( 65)       0
  24    1    4   0.0000000   ( 23)       3     0.00000   ( 45)      12     0.00000   ( 66)       0
 ---------------------------------------------------------------------------------------------------
0ILLEGAL BOND LENGTH ON Z MATRIX CARD   2
0ILLEGAL BOND LENGTH ON Z MATRIX CARD   3
0ILLEGAL BOND LENGTH ON Z MATRIX CARD   4
0ILLEGAL BOND LENGTH ON Z MATRIX CARD   5
0ILLEGAL BOND LENGTH ON Z MATRIX CARD   6
0ILLEGAL BOND LENGTH ON Z MATRIX CARD   7
0ILLEGAL BOND LENGTH ON Z MATRIX CARD   8
0ILLEGAL BOND LENGTH ON Z MATRIX CARD   9
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  10
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  11
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  12
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  13
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  14
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  15
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  16
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  17
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  18
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  19
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  20
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  21
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  22
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  23
0ILLEGAL BOND LENGTH ON Z MATRIX CARD  24
0ATOM 3 CANNOT LIE ON THE Z AXIS


 ------------------------------------------------------------------------------------------------------------------------------------
                    COORDINATES IN ANGSTROMS                               COORDINATES IN BOHR                    ATOMIC ISOTOPE/MASS
   I   AN        X             Y             Z                        X             Y             Z               MASS#    MASS (AMU)
 ------------------------------------------------------------------------------------------------------------------------------------
   1    6    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000             12    12.00000000
   2    6    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000             12    12.00000000
   3    6    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000             12    12.00000000
   4    6    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000             12    12.00000000
   5    6    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000             12    12.00000000
   6    6    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000             12    12.00000000
   7    6    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000             12    12.00000000
   8    6    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000             12    12.00000000
   9    6    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000             12    12.00000000
  10    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  11    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  12    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  13    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  14    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  15    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  16    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  17    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  18    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  19    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  20    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  21    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  22    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  23    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
  24    1    0.0000000     0.0000000     0.0000000                0.0000000     0.0000000     0.0000000              1     1.00782504
 ------------------------------------------------------------------------------------------------------------------------------------
0RUN TERMINATED DUE TO Z MATRIX ERROR(S)

0*** DUE TO AN UNFORTUNATE ERROR, THIS CALCULATION HAS BEEN TERMINATED     ... 5 ... 4 ... 3 ... 2 ... 1 ...
1

                                                            .AXXXXXXIIXXXXXXA.
                                                         XXXXXXXXXAAAAAAAAAXXXXXXXA
                                                      AXXXXXV..................VXXXXXA
                                                   AXXXV..........................VXXXA.
                                                  AXXXV.............................VXXXXXXA
                                           .....AXXXV................................VXX....XXA
                                          AXXXXXXXXV..................................XXX.....VXXA
                                         AXXXV....VA.................................AXV........VXXXXXXX...
                                        AXXXV......VA...............................AV...........VATTTTTTTVXXXA
                            .......   ./AXXV........VA............................TT..............O..........TVXA
                         .AXXXXXXXXXA...XXV..........T......................../A.................O.............VXA
                        AXXVTTTTTTTXXVTXXVT............LLVL................AXXV.................O...............VXA.
                      AXXXV.......DXV..VA.................VXXXA-.....///AAXXV..............LL...................IXXA
                     AXXV........AV.........................TTTTVXXXATTT/...................VA................./AXXX
                    /AXV.........VX.........................................................XXA................AXXTXD
                    AXXV.........IX..AV.......................             ..................VO.............2XXV..XB-
                    XX4A..........DOAV........................ WHAT A BOMB ..................XB....L...JAVXXXV..../X-
                    -X.4XL.........EO.........................             .................OB......TTTTT/........3XE
                   3X/...TXXLLL.....O..........................................,O,.........OO....................AXV
                    VX.............OS,.......................................,BO.........BBO..................../XXD
                    VXA............,OL.....................................OXB........JXP......................AXXV
                     VXA.............OO..........,O,....................JXXXXXX./...................6..........AXXXV
                      VATTT...........OAO,...,,,O,BO...............,,-JXXXXXX./XLX,.............../XA.......,AXXXV
                      3XXV..............TTAOOTT...BXXL-JVXXXXXXXXXXXXXXXXXXXXXXXXA-,...VXV.........VXL.../AXXV
                        JXV..............,,OO.....AXXXVTTTTTTTTTTTTTT......XXX..4XXXXXLXV............VXXXXV
                         VXXXV.VAXXXXXXXXXXXX..XXXX.......XXX..............XXX...3OOOOOXXXXXXXXXXXXXXXXXV
                             VXXXXXXXXXXXXXXXXXXX.........XXX..............XXXXXXXXXXXXXXXXXXXXXXXXVT
                                              TVXXXXXXXXXXXXX..............XXXVT
                                                          XXX..............XXX
                                                          XXX..............XXX                        .II.
                             .IIII.              ...II....XXX..............XXX            .........IIIIIIIIIIIIII....
                .IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII          IIIIIIIII
                III                                                       IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
                 TIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIITTTTTXXX                                 TIT
                                                         XXX................XXX
                                           .            XXX.................XXXCCC..CCCCC.
                                       .CCCCCCCCC...CCCXXXX.................XXXX....CCCC..CCC.
                                     .CCCCTTCCCCC.CC....XXXX.................XXXX....CC.....CCC.
                                     .DTCCC......CCCCCCCCCCL........,,JCCCCCCCCCCCCCCC...CCCCC.
                                      .   CCC...........CCCCCCCCCCCCCCCCCT.TCCCCC....CCCC    .
                                            CCCCCC..................................1HC)
                                                CCCC..............................CCCC
          GGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGG
0EVEN IF YOU ARE ON THE RIGHT TRACK,
 YOU WILL GET RUN OVER IF YOU JUST SIT THERE.

0*** ABNORMAL TERMINATION, TOTAL CPU TIME FOR THIS CALCULATION WAS     0.000 MINUTES.
